static/sass/components/_post-header.scss

```diff
@@ -2,22 +2,58 @@
 // ===========
 
 .title {
-  @include title;
   margin: size('gutter') auto;
   max-width: size('full-content');
   padding: 0 size('gutter');
 
-  &:first-child {
+  .no-image & {
     @include text-shadow;
+    @include title;
   }
+}
+
+// Titles on pages that live inside title wrapper
+.title-main {
+  @include text-shadow;
+  @include title;
+}
+
+// Titles on pages that don't have title wrapper
+.page-title {
+  @include text-shadow;
+  @include title;
+}
+
+.title-post-type {
+  @include font-family('body');
+  background-color: color('invert');
+  color: color('background');
+  display: inline-block;
+  font-weight: 400;
+  margin-left: negative('gutter');
+  padding: size('quarter-shim') size('gutter');
+  position: relative;
 
-  em {
-    @include font-family('body');
-    font-size: calc(1.25rem + 1.75vw);
-    font-style: italic;
+  @include after('')  {
```

Comment:
Using `clip` instead of css triangles would allow us to use this over-top the watercolor and other backgrounds.